sale_product_set/models/product_set.py

```diff
@@ -19,3 +19,7 @@ class ProductSet(models.Model):
         'product_set_id',
         string="Products"
     )
+    image = fields.Binary(
+        string='Image',
+        help='Select an image to display for the product set'
```

Comment:
Please add `attachment=True` for having the advantages it brings

Reply:
@pedrobaeza Thanks for the hint, I applied the change.